src/tests/roc_audio/test_resampler.cpp

```diff
@@ -149,6 +149,38 @@ TEST(resampler, upscaling_twice_single) {
     }
 }
 
+#ifdef ROC_TARGET_DARWIN
+// Check upsampling quality and the cut-off band with white noise.
+IGNORE_TEST(resampler, upscaling_twice_awgn) {
```

Comment:
Why to copy the whole test? It's enough to copy only the header:

```
// Check upsampling quality and the cut-off band with white noise.
#ifdef ROC_TARGET_DARWIN
IGNORE_TEST(resampler, upscaling_twice_awgn) {
#else
TEST(resampler, upscaling_twice_awgn) {
#endif
    ...
}
```

Reply:
fixed